docs/source/azure/configuration/acitutorial.rst

```diff
@@ -112,7 +137,7 @@ with the following command:
 
```

Comment:
Line 131: I think the @ sign is a typo in `--parameters @c7n-aci-tutorial.json`

Reply:
I think the `@` sign indicates to the [azure cli](https://docs.microsoft.com/en-us/cli/azure/group/deployment?view=azure-cli-latest#az-group-deployment-create) that it is a path to a parameters file.